lib/scripts/persist_procedures.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require "csv"
+require_relative "create_cbhpm_procedures"
+
+module Scripts
+  class PopulateProcedures
+    attr_reader :path_file
+
+    def initialize(path_file)
+      @path_file = path_file
+    end
+
+    def run!
+      populate_database!
+    end
+
+    private
+
+    def populate_database!
+      ActiveRecord::Base.transaction do
+        procedures.each do |procedure|
+          next if procedure_persisted?(procedure)
+
+          procedure_instance = procedure_build(procedure[:code], procedure[:name])
+
+          save_procedure!(procedure_instance, procedure[:port], procedure[:anesthetic_port])
+        end; nil
+      rescue StandardError => e
+        raise StandardError, e.message
+      end
+    end
+
+    def save_procedure!(procedure_instance, port, anesthetic_port)
+      success_message = success_message(procedure_instance.code)
+
+      puts_message(success_message)
+
+      create_cbhpm_procedures(procedure_instance, port, anesthetic_port) if procedure_instance.save!
+    rescue StandardError => e
+      fail_message = fail_message(procedure_instance.code, e)
+
+      logger.warn(fail_message)
```

Comment:
maybe `log.error`?

Reply:
I preferred to remove logger.warn and let only raise error. This current [PR](https://github.com/espoo-dev/med_system_backend/pull/183/commits/fab92b0a6a9c794ff1d20321c63b85f94fb1effa) was merged, so I created a new PR and this is the commit [link](https://github.com/espoo-dev/med_system_backend/pull/183/commits/fab92b0a6a9c794ff1d20321c63b85f94fb1effa)